controllers/openstackmachine_controller.go

```diff
@@ -334,11 +335,16 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, logger
 	case infrav1.InstanceStateActive:
 		logger.Info("Machine instance is ACTIVE", "instance-id", instance.ID)
 		openStackMachine.Status.Ready = true
-	case infrav1.InstanceStateBuilding:
-		logger.Info("Machine instance is BUILDING", "instance-id", instance.ID)
-	default:
+	case infrav1.InstanceStateBuild:
```

Comment:
This should be deleted, because VM with BUILD status is on the way to ACTIVE,  thus should go into Requeue.

Reply:
I agree, although the additional logging is nice. I wonder if we could add a generic log message to the default case below?